internal/core/cobra_utils_test.go

```diff
@@ -71,6 +88,25 @@ func Test_handleUnmarshalErrors(t *testing.T) {
 	}))
 }
 
+func Test_RawArgs(t *testing.T) {
```

Comment:
shoudn't we have a `testcobraRun` func that only test `cobraRun` ? 

I find it strnage that we hav to test an entire command. 

Shouldn't we only call `cobraRun()` in the test.

Reply:
we could :smile: I'll fix that

Reply:
Though it may be more complicated :thinking: